src/Styles/App.module.css

```diff
@@ -1,16 +1,21 @@
 .navbar {
```

Comment:
- [ ] The nav bar has a profile icon, kindly add it to your design.  you should replicate the Zeplin design

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220609_193055](https://user-images.githubusercontent.com/25227022/172919520-257256d5-59c2-4ce2-a07d-cfde87531da6.png)|![Screenshot_20220609_193441](https://user-images.githubusercontent.com/25227022/172919714-776c1886-f47b-4791-94af-446781295b19.png)|